charts/sshpiper/templates/_helpers.tpl

```diff
@@ -48,4 +48,16 @@ Selector labels
 {{- define "sshpiper.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "sshpiper.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "sshpiper.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
```

Comment:
serviceAccount is a must
sshpiper is watching all crds and load secret with it

Reply:
Rationale:
- yes and that's why it's enabled by default
- it's common practice to leave creating **new** service account optional, because user can already have its own she wants to use
- user's custom service account can be even more restricted, e.g. accessing only Pipes, but not Secrets (if no pulibc key auth. is expected)

I can drop if you would insist.

Reply:
Thanks, new knowledge learned 
https://helm.sh/docs/chart_best_practices/rbac/